generate/beat/{{cookiecutter.beat}}/beater/{{cookiecutter.beat}}.go

```diff
@@ -41,26 +42,14 @@ func (bt *{{cookiecutter.beat|capitalize}}) Config(b *beat.Beat) error {
 
 func (bt *{{cookiecutter.beat|capitalize}}) Setup(b *beat.Beat) error {
 
-	// Setting default period if not set
-	if bt.beatConfig.{{cookiecutter.beat|capitalize}}.Period == "" {
-		bt.beatConfig.{{cookiecutter.beat|capitalize}}.Period = "1s"
-	}
-
 	bt.client = b.Publisher.Connect()
-
-	var err error
-	bt.period, err = time.ParseDuration(bt.beatConfig.{{cookiecutter.beat|capitalize}}.Period)
-	if err != nil {
-		return err
-	}
-
 	return nil
 }
 
 func (bt *{{cookiecutter.beat|capitalize}}) Run(b *beat.Beat) error {
 	logp.Info("{{cookiecutter.beat}} is running! Hit CTRL-C to stop it.")
 
-	ticker := time.NewTicker(bt.period)
+	ticker := time.NewTicker(bt.config.{{cookiecutter.beat|capitalize}}.Period)
```

Comment:
yeah, with ucfg we can separate and pass the beat config section to init method. Will add item to #2005